.readthedocs.yaml

```diff
@@ -0,0 +1,15 @@
+version: 2
+
+# Build from the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# Explicitly set the version of Python and its requirements
+python:
+  version: 3.8
```

Comment:
Shouldn't python version 3.6 and 3.7 also be supported?

Reply:
This only specifies the Python version used to build the docs, so I went for the most recent supported version.